tpl/programs.json

```diff
@@ -1222,6 +1222,10 @@
         "file": "temperature-ds18b20.js",
         "title": "Thermometer - DS18B20"
       },
+      {
+        "file": "temperature-dual-ds18b20.js",
+        "title": "Thermometer - Dual DS18B20"
```

Comment:
Did you build this circuit and test the program? (I should've been asking this for all the ones you've added)

Reply:
No, but just to be clear, and I didn't add the JS file. I just added it to programs.json so the md would be generated.

Reply:
@dtex I think the reason I had removed it from the list is because I wasn't positive that it worked correctly and reliably (but I can't remember the specifics)